go/cmd/vttablet/vttablet.go

```diff
@@ -29,11 +29,12 @@ import (
 )
 
 var (
-	tabletPath     = flag.String("tablet-path", "", "tablet alias")
-	enableRowcache = flag.Bool("enable-rowcache", false, "enable rowcacche")
-	overridesFile  = flag.String("schema-override", "", "schema overrides file")
-	tableAclConfig = flag.String("table-acl-config", "", "path to table access checker config file")
-	lockTimeout    = flag.Duration("lock_timeout", actionnode.DefaultLockTimeout, "lock time for wrangler/topo operations")
+	tabletPath            = flag.String("tablet-path", "", "tablet alias")
+	enableRowcache        = flag.Bool("enable-rowcache", false, "enable rowcacche")
+	enforceTableACLConfig = flag.Bool("enforce-tableacl-config", false, "if this flag is true, vttablet will fail to start if a valid tableacl config does not exist")
```

Comment:
nit: Move this line two lines down directly below "table-acl-config" such that related flags are at the same location.